### ISSUE TITLE
Attempt awspec compatibility

### ISSUE THREAD

**Issue by clintoncwolfe** [inspec/inspec#3641]
### 🎛 Description

🙋 feature request

Let's try to make a bridge to the awspec kit, so InSpec can leverage the resources.

**Comment by skpaterson** [comment]
Hi @clintoncwolfe - are you still targeting this work for the InSpec 4.0 release?

**Comment by clintoncwolfe** [comment]
No, we're not.  I did some spike work on it in https://github.com/inspec/inspec/tree/cw/awspec - I got to the point where it was clear we'd have to alter the inheritance tree (both awspec resources and inspec resource demand to dictate the superclass).

Not impossible, and the payoff is large!

**Comment by skpaterson** [comment]
Thanks @clintoncwolfe will migrate this across in that case.
